src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -675,6 +676,13 @@ public Block get(final PlayerMoveEvent e) {
 				return EvtMoveOn.getBlock(e);
 			}
 		}, 0);
+		// PlayerItemDamageEvent
+		EventValues.registerEventValue(PlayerItemDamageEvent.class, ItemStack.class, new Getter<ItemStack, PlayerItemDamageEvent>() {
+			@Override
+			public ItemStack get(PlayerItemDamageEvent event) {
```

Comment:
Shouldn’t you convert the ItemStack to an ItemType and return that instead?

Reply:
Ahh.... dur! totally slipped my mind, thank you!

Reply:
Fixed :)